tests/integration/static_ip_address_test.py

```diff
@@ -374,6 +375,10 @@ def test_add_static_ipv6_with_min_state(eth2_up):
     assertlib.assert_state(desired_state)
 
 
+@pytest.mark.xfail(
```

Comment:
I would prefer to see this in a separate commit that explains why it is needed.
It is also a bit dangerous, as this is covering also cases where we expect this to exist. Maybe we can add a condition on when this is acceptable? Like, the OS version? (maybe we have an env variable for that?)